webpack.config.babel.js

```diff
@@ -31,11 +32,31 @@ const styleBaseLoaders = [
   `less?sourceMap=${CSS_MAPS}`
 ];
 
+const PROD_CONFIG = {
```

Comment:
Maybe we should have a separate config file where we store the configurations if these constants increase